### ISSUE TITLE
Use of the challenge dataset

### ISSUE THREAD

**Issue by Adele0108** [wongzbb/DiffMa-Diffusion-Mamba#1]
Hi, do you use all the training dataset of the challenge for training, and extract part of it for testing? What is the ratio of the training set to the test set, and do you use the validation dataset of the challenge?

**Comment by wongzbb** [comment]
> 嗨，您是否将挑战的所有训练数据集用于训练，并提取其中的一部分进行测试？训练集与测试集的比率是多少，是否使用挑战的验证数据集？

Yes, we only used the entire training dataset from the challenge, with a split ratio of 8:2, and we didn't use the validation dataset. You can download and check out the data we processed for more details.

**Comment by tuhanglsWorld** [comment]
I'd like to ask why I didn't see the separate test section in your brain dataset?
